composer.json

```diff
@@ -27,7 +27,8 @@
     },
     "require-dev": {
         "dg/bypass-finals": "^1.3",
-        "symfony/phpunit-bridge": "^5.4|^6.0"
+        "symfony/phpunit-bridge": "^5.4|^6.0|^7.0",
+        "phpunit/phpunit": "^9.0|^10.0"
```

Comment:
And so this can be reverted - that's a feature of PHPUnit bridge, it allows you to get rid of PHPUnit dependency directly